### ISSUE TITLE
Not really an issue but a request - Terminal control

### ISSUE THREAD

**Issue by critycal** [njfaria/dalylog#9]
Hello,

Is it possible to run and start DalyLog from the terminal without any IDE? I feel like it would be more reliable and not require the resources to run a display and vnc for tiny devices.

**Comment by SunnyPerth** [comment]
Yes, this would be a fantastic addition! Even if it means that we run it on the qui to setup and then restart with a flag (like Dalylog -No_IDE)

**Comment by PanicMan** [comment]
+1 👍

**Comment by SunnyPerth** [comment]
Just bumping this request. I might need to move Daly log to a pi-zero, not the best pi to run a full desktop on. Hence hoping we can get a command line version of the tool. It “should” be reasonable easy to convert the code to run command line? Maybe a flag that tells dalylog to run on command line, with a second flag to tell what configuration file to use?
